graftm/db_search_results.py

```diff
@@ -19,6 +19,6 @@ def search_objects(self): # Return SequenceSearchResult object with parameters d
     def hit_count(self): # Return the number of hits found
         return self.hit_count  
     
-    def slash_endings(self): # Return True if sequence headers end in /1 or /2, False if not
-        return self.slash_endings
+    def has_slash_endings(self): # Return True if sequence headers end in /1 or /2, False if not
```

Comment:
I think this method is superfluous, because `has_slash_endings` is accessible from any code that has this object. So I would remove it.